packages/pn-personagiuridica-webapp/src/App.tsx

```diff
@@ -124,19 +125,33 @@ const App = () => {
     }
   }, [sessionToken]);
 
-  const notificationMenuItems: Array<SideMenuItem> = [
-    {
-      label: t('menu.notifiche'),
-      route: routes.NOTIFICHE,
-    },
-  ];
+  const mapDelegatorSideMenuItem = (): Array<SideMenuItem> | undefined => {
+    // if the current user is not a groupAdmin can also see own PG notifications,
+    // else it sees only delegated notifications and we return undefined
+    if (!isGroupAdmin) {
```

Comment:
I'm not sure about this check. If an user isn't a groupAdmin and he doesn't have mandates, does he see both menu voices?